src/twister2/specification_processor.py

```diff
@@ -290,6 +291,13 @@ def should_skip_for_spec_type_unit(test_spec: YamlTestSpecification, platform: P
     return False
 
 
+def should_skip_for_env(test_spec: YamlTestSpecification, platform: PlatformSpecification) -> bool:
+    if not platform.env_satisfied:
+        _log_test_skip(test_spec, platform, 'environment ({}) not satisfied'.format(', '.join(platform.env)))
```

Comment:
I know, that this message was copied from v1, but what do you think about changing it into something like:
`environment variables ({}) not set`? For me this is more meaningful for potential user.